rust/otel-arrow-rust/src/lib.rs

```diff
@@ -33,3 +33,6 @@ pub mod pdata;
 pub mod proto;
 
 pub use decode::decoder::Consumer;
+
+// Debug tests for EncodedLen visitor functionality
+//pub mod debug_test;
```

Comment:
I'm not sure if the intention was to have these `debug_test.rs` & `debug_detailed.rs` files checked in? If these are just for your local debugging during development, maybe we can remove them?

If the intention here is just to have some local entrypoint into the codebase for debugging/testing, I'll also mention that you can add rust files to `rust/otel-arrow-rust/src/bin/`, and inside here you can add any rust files you want. They can have a main funciton, and/or tests.

For example if you added `rust/otel-arrow-rust/src/bin/debug_test.rs` you could run your tests/main using
```
cargo test --bin debug_test
cargo run --bin debug_test
```
What's nice is that this folder is also covered by `.gitignore`, so there's no risk of accidently checking in this scratch code

Reply:
I mean to erase these tests, they were transient.